types/port-numbers/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for port-numbers 5.0
+// Project: https://github.com/silverwind/port-numbers#readme
+// Definitions by: Alex Bradley <https://github.com/abradley>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface ServiceInfo {
```

Comment:
The [source code](https://github.com/silverwind/port-numbers/blob/master/index.js#L20) reuses an existing object. How do you look at marking all fields as `readonly`?

Reply:
Sure, that makes sense. Done.